scripts/release.js

```diff
@@ -114,27 +119,49 @@ fs.writeFileSync(
 spawnOrFail('npm', [`version ${versionString} --no-git-tag-version`]);
 logger.log(`Updated package.json version to ${versionString}`);
 
-const updatedSdkVersion = spawnOrFail('npm', [`show amazon-chime-sdk-js version`]).trim();
+const updatedSdkVersion = spawnOrFail('npm', [
+  `show amazon-chime-sdk-js version`,
+]).trim();
 
 // Skip updating peer dependencies for hotfix
 if (release_option !== '5') {
   // Update the peer dependency to the most updated version of the SDK
-  logger.log(`Installing SDK Version: ${updatedSdkVersion} into the meeting demo, chat demo, and as a peerDependency and devDependency of the react library.`);
+  logger.log(
+    `Installing SDK Version: ${updatedSdkVersion} into the meeting demo and as a peerDependency and devDependency of the react library.`
+  );
 
-  let componentsPackageJson = JSON.parse(fs.readFileSync('package.json', 'utf-8'));
-  componentsPackageJson.peerDependencies['amazon-chime-sdk-js'] = `^${updatedSdkVersion}`;
-  componentsPackageJson.devDependencies['amazon-chime-sdk-js'] = `^${updatedSdkVersion}`;
+  let componentsPackageJson = JSON.parse(
+    fs.readFileSync('package.json', 'utf-8')
+  );
+  componentsPackageJson.peerDependencies[
+    'amazon-chime-sdk-js'
+  ] = `^${updatedSdkVersion}`;
+  componentsPackageJson.devDependencies[
+    'amazon-chime-sdk-js'
+  ] = `^${updatedSdkVersion}`;
 
   fs.writeFileSync(
     'package.json',
     JSON.stringify(componentsPackageJson, null, 2)
   );
 
-  logger.log("NPM Installing component library...")
+  logger.log('NPM Installing component library...');
   spawnOrFail('npm', [`install`]);
 
-  // Udpate meeting demo to the most up to date version of the SDK
-  process.chdir(path.join(__dirname, '../demo/meeting'));
+  // Checkout the meeting demo and udpate the demo to the most up to date version of the SDK
```

Comment:
Can we move this part down to before we prepare the release URL at the end?

Reply:
Sure, and we should still keep the `release_option !== '5'` as a condition?